### ISSUE TITLE
descriptor is how to form?

### ISSUE THREAD

**Issue by wkwukai** [jiexiong2016/GCNv2_SLAM#8]
Hello,can you tell me ,after upsample,and binary neural networ,form an 256x320x240 feature map/heatmap ,and now,how to operation?

**Comment by jiexiong2016** [comment]
The detection decoder outputs a mask which can be reshaped to the original image resolution using pixel shuffling. By taking the coordinate of keypoints, a bilinear sampling can be performed using "torch.nn.functional.grid_sample". Please check the pytorch documents for more details.
